outputs.tf

```diff
@@ -1,13 +1,49 @@
-# Output value definitions
-# Define all output values that other modules or configurations might need
-#
-# Example:
-# output "instance_ip" {
-#   description = "The IP address of the compute instance"
-#   value       = google_compute_instance.example.network_interface[0].access_config[0].nat_ip
-# }
-#
-# output "instance_name" {
-#   description = "The name of the compute instance"
-#   value       = google_compute_instance.example.name
-# }
+output "cluster_name" {
+  description = "The name of the EKS cluster"
+  value       = aws_eks_cluster.main.name
+}
+
+output "cluster_arn" {
+  description = "ARN of the EKS cluster"
+  value       = aws_eks_cluster.main.arn
+}
+
+output "cluster_endpoint" {
```

Comment:
This doesn't output the cluster_certificate_authority_data or cluster_oidc_issuer_url which we will need for getting the kubeconfig